docs/vanilla.md

```diff
@@ -0,0 +1,144 @@
+# Velero Plugin for vSphere in Vanilla Kubernetes Cluster
+
+## Table of Contents
+1. [Compatibility](#compatibility)
+2. [Prerequisites](#prerequisites)
+3. [Install](#install)
+4. [Uninstall](#uninstall)
+5. [Backup](#backup)
+6. [Restore](#restore)
+
+## Compatibility
+* Velero - Version 1.3.2 or above
```

Comment:
Should be 1.5.1 or above as this page covers plugin 1.1.0.
We can add a note to refer to the older README for 1.0.2 plugin

Reply:
Will change to 1.5.1. 

Since the `Latest Release` was mentioned in the README.md, I plan not to mention it in vanilla doc.

Reply:
That's fine.